apps/ledger-live-desktop/src/renderer/components/PostOnboardingHub/PostOnboardingHubBanner.tsx

```diff
@@ -24,7 +24,7 @@ const PostOnboardingHubBanner = () => {
   const navigateToPostOnboardingHub = useNavigateToPostOnboardingHubCallback();
 
   const handleNavigateToPostOnboardingHub = useCallback(() => {
-    track("button_clicked", { button: "What’s next for your Ledger Stax" });
+    track("button_clicked", { button: "What’s next for your device" });
```

Comment:
Not my/our call, but wouldn't it be interesting to have a property `deviceModel` to our events ?

Reply:
I think it's automatically added to additional event properties based on the last connected device